packages/@n8n/nodes-langchain/nodes/tools/ToolWorkflow/ToolWorkflow.node.ts

```diff
@@ -36,6 +37,7 @@ export class ToolWorkflow extends VersionedNodeType {
 			1.2: new ToolWorkflowV1(baseDescription),
 			1.3: new ToolWorkflowV1(baseDescription),
 			2: new ToolWorkflowV2(baseDescription),
+			2.1: new ToolWorkflowV2_1(baseDescription),
```

Comment:
Do we need a separate module for this? We could just set `returnAllItems` to true if `this.getNode().typeVersion > 2.0`

Reply:
I want to avoid doing this in workflowtoolservice, but now realizing could have done this at the module level. Will refactor.